src/commands/deploy.js

```diff
@@ -4,7 +4,7 @@ const path = require('path')
 const chalk = require('chalk')
 const { flags } = require('@oclif/command')
 const get = require('lodash.get')
-const fs = require('fs')
+const fs = require('fs-extra')
```

Comment:
So we can switch to `async` interfaces

Reply:
Could we use `utils.promisify` instead?
`fs-extra` includes `graceful-fs` which monkey patches global variables.

Reply:
Sure, I can also open an issue to cleanup `fs-extra` from the codebase

Reply:
See ef9a68ae34cf4357d2bff859671fd22db3c7fdc1 and See https://github.com/netlify/cli/issues/1241

Reply:
Oh I did not know we used it elsewhere :)